lib/collections/addon/components/collections-submission/component.ts

```diff
@@ -188,6 +188,13 @@ export default class Submit extends Component {
             }
             await this.collectionSubmission.save();
 
+            const resubmitAction = this.store.createRecord('collection-submission-action', {
+                actionTrigger: CollectionSubmissionActionTrigger.Resubmit,
+                comment: 'Resubmitting',
```

Comment:
Is this always supposed to be 'Resubmitting', or is this supposed to be taken from the comment field?

Reply:
There is no comment field (but maybe there will be in the future?), so I will remove this